specs/subresourceintegrity/spec.markdown

```diff
@@ -247,15 +247,15 @@ resource in order to provide agility in the face of future discoveries.
 For example, the "Hello, world." resource described above may be described
```

Comment:
Now that the resource is no longer "Hello, world.", perhaps we should rephrase that and just refer to "the resource described in the previous section may be described..."